### ISSUE TITLE
Request: Webhook path flexibility

### ISSUE THREAD

**Issue by dgarozzo** [FlyingDiver/Indigo-HTTPd-2#6]
Thanks for this plugin. I'd like to use it for a project I'm working on. Unfortunately, the web hook I need to use for it doesn't send requests to /webhook. Could you make two minor changes for me? Can you change the line (in two places):

 elif request.path.startswith("/webhook"):

to:

elif "webhook" in request.path:

This should provide the same functionality, and allow other web hooks to take advantage of this plugin.

**Comment by dgarozzo** [comment]
Forked and created a pull request for this.

**Comment by FlyingDiver** [comment]
Can you post the debug output of the modified plugin, when it's handling a request with your web hook?  I want to make sure that the broadcast message it's sending out is unique.

**Comment by dgarozzo** [comment]
I think this is working as you hope/expect?

```
Oct 18, 2020, 11:17:46 AM
   HTTPd 2 Debug                   MyRequestHandler: GET to port 5555 from 192.168.1.251:49696 for /bouncie-webhook
   HTTPd 2 Debug                   MyRequestHandler: No password specified in device configuration, skipping authentication
   HTTPd 2 Debug                   do_webhook query = , path = /bouncie-webhook
   HTTPd 2 Debug                   Webhook to httpd_bouncie-webhook = {"headers": {"accept-language": "en-us", "accept-encoding": "gzip, deflate", "connection": "keep-alive", "accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1.2 Safari/605.1.15", "host": "192.168.1.240:5555", "cookie": "--redacted--", "upgrade-insecure-requests": "1"}, "request": {"path": "/bouncie-webhook", "client": "192.168.1.251", "command": "GET"}, "payload": null, "vars": {}}
   HTTPd 2 Debug                   MyRequestHandler: GET to port 5555 from 192.168.1.251:49707 for /webhook
   HTTPd 2 Debug                   MyRequestHandler: No password specified in device configuration, skipping authentication
   HTTPd 2 Debug                   do_webhook query = , path = /webhook
   HTTPd 2 Debug                   Webhook to httpd_webhook = {"headers": {"accept-language": "en-us", "accept-encoding": "gzip, deflate", "connection": "keep-alive", "accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1.2 Safari/605.1.15", "host": "192.168.1.240:5555", "cookie": "--redacted--", "upgrade-insecure-requests": "1"}, "request": {"path": "/webhook", "client": "192.168.1.251", "command": "GET"}, "payload": null, "vars": {}}
   HTTPd 2 Debug                   MyRequestHandler: GET to port 5555 from 192.168.1.251:49708 for /favicon.ico
   HTTPd 2 Debug                   MyRequestHandler: No password specified in device configuration, skipping authentication
   HTTPd 2 Debug                   MyRequestHandler: Unknown GET request: /favicon.ico
   HTTPd 2 Debug                   MyRequestHandler: GET to port 5555 from 192.168.1.251:49716 for /test-webhook
   HTTPd 2 Debug                   MyRequestHandler: No password specified in device configuration, skipping authentication
   HTTPd 2 Debug                   do_webhook query = , path = /test-webhook
   HTTPd 2 Debug                   Webhook to httpd_test-webhook = {"headers": {"accept-language": "en-us", "accept-encoding": "gzip, deflate", "connection": "keep-alive", "accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1.2 Safari/605.1.15", "host": "192.168.1.240:5555", "cookie": "--redacted--", "upgrade-insecure-requests": "1"}, "request": {"path": "/test-webhook", "client": "192.168.1.251", "command": "GET"}, "payload": null, "vars": {}}
   HTTPd 2 Debug                   MyRequestHandler: GET to port 5555 from 192.168.1.251:49717 for /favicon.ico
   HTTPd 2 Debug                   MyRequestHandler: No password specified in device configuration, skipping authentication
   HTTPd 2 Debug                   MyRequestHandler: Unknown GET request: /favicon.ico

```

**Comment by FlyingDiver** [comment]
Yeah, that looks fine.  Published 2.1.2.
